easybuild/easyblocks/h/hadoop.py

```diff
@@ -62,6 +70,17 @@ def install_step(self):
         else:
             super(EB_Hadoop, self).install_step()
 
+    def post_install_step(self):
+        """After the install, copy libsnappy into place."""
+        for native_library, lib_path in self.cfg['extra_native_libs']:
+            lib_root = get_software_root(native_library)
+            if not lib_root:
+                self.log.warn("%s not found. Skipping install" % native_library)
+                continue
```

Comment:
no, fail if you were asked to include extra native libraries with the required dep missing

```python
    raise EasyBuildError("Dependency %s is required for specified extra native libs, but not found" % native_library)
```